src/utilities/permissions.ts

```diff
@@ -762,6 +762,9 @@ const queryPermissions = {
       isUserAdmin(user) || (getPermission(['update_expansion_rule_by_pk'], user) && isUserOwner(user, expansionRule))
     );
   },
+  UPDATE_PLAN: (user: User | null, plan: PlanWithOwners): boolean => {
```

Comment:
Part of the refactor that I plan on making is having the permission keys 1-to-1 with the GQL object. Do you mind, updating this to match the key of the query that does the updating for plan tags if it doesn't already, please?

Reply:
What do you mean exactly? Like `CREATE_PLAN_TAGS` instead of a general "can update plan"?

Reply:
Right, that object is more for getting the permission for the query, rather than getting a permission for the "feature"